src/main/java/Main.java

```diff
@@ -7,7 +7,9 @@ public static void main(String[] args) {
 
         while (!grandPrix.isFinished()) {
             grandPrix.play();
-            OutputView.printResult(grandPrix);
+            OutputView.printPosition(grandPrix);
         }
+        System.out.println();
+        OutputView.printChampion(grandPrix.report());
```

Comment:
```suggestion
        OutputView.printChampion(grandPrix.report());
```

`System.out.println();`은 출력 메소드 안으로 숨겨볼 수 있을 것 같네요 😄